package.json

```diff
@@ -8,11 +8,12 @@
     "node": ">=22"
   },
   "scripts": {
-    "pretest": "npm run test:lint",
```

Comment:
Why is this gone?

Reply:
I moved it to line 16 so that it was in alphabetical order 😁